### PR TITLE
Legacy Cleanup

### PR DESCRIPTION
**Pull request by noslouch**
Updates for better legacy compat. Depends on nypublicradio/publisher#6.

### CODE REVIEW

File: app/lib/is-js.js [hunk 1]
```diff
@@ -2,5 +2,8 @@ export default function isJavascript(scriptTag) {
   // TODO: add a console warning if the script tag doesn't have an attribute? 
   // seems like it's required for some parts of ember consumption
   let type = scriptTag.attributes.type ? scriptTag.attributes.type.value : 'text/javascript';
-  return /(?:application|text)\/(deferred-)?javascript/i.test(type);
+  // guard against chrome-extension:// scripts
+  let badSrc = scriptTag.src && !scriptTag.src.match(/^http|^\//);
```
Comment (mike-hearn) [inline app/lib/is-js.js]: Just checking on this, there's no worry of a relative import being used?

File: app/components/site-chrome/template.hbs [hunk 2]
```diff
@@ -109,23 +109,23 @@
           {{/if}}
         </li>
 
-        {{#link-to 'stream' tagName='li' href=false class="list-item list-item--nav" }}
+        {{#link-to 'stream' tagName='li' href=false class="list-item list-item--nav"}}
           {{link-to 'All Streams' 'stream' class="inherit" preventDefault=false}}
         {{/link-to}}
 
-        {{#link-to 'schedule' current-when="schedule.date" tagName='li' href="false" class="list-item list-item--nav" }}
+        {{#link-to 'schedule' current-when="schedule.date" tagName='li' href=false class="list-item list-item--nav"}}
```
Comment (mike-hearn) [inline app/components/site-chrome/template.hbs]: Minor quibble slash question for my own future coding: do we have a rule in our codebase for when to use singe vs. double quotes? It looks inconsistent in this file, for example `current-when` on this line vs. L120.

Comment (noslouch) [inline app/components/site-chrome/template.hbs]: there's no rule, but I prefer single quotes in all cases and escaping apostrophes in longer strings, but that's me.

File: app/lib/is-js.js (hunk 1 above)
Comment (noslouch) [inline app/lib/is-js.js]: not sure what you mean here

Comment (mike-hearn) [inline app/lib/is-js.js]: I was just wondering if there would ever be a case where src would be relative to the HTML its being used in, e.g. <script src="./bundle.js">, which would not match either `^http` or `^/`, but would still potentially be valid JS.

Comment (noslouch) [inline app/lib/is-js.js]: ah, yeah. THAT IS A GOOD CATCH.

Comment (mike-hearn) [inline app/lib/is-js.js]: I DID IT

**Pull request merged**